updateLocalesHelper/updateLocaleStatus.py

```diff
@@ -1,59 +1,92 @@
 #! /bin/env python
-import json
+from configparser import ConfigParser
+from http import HTTPStatus
+from json import dump
```

Comment:
I prefer to `import json`, because it's clearer to read `json.dump` than just `dump`. Also, that's how they do it in the examples: https://docs.python.org/3.7/library/json.html